flink-table/flink-table-common/src/main/java/org/apache/flink/table/types/logical/StructuredType.java

```diff
@@ -294,7 +294,7 @@ public StructuredType build() {
 
     private final boolean isInstantiable;
 
-    private final StructuredComparision comparision;
+    private final StructuredComparison comparision;
```

Comment:
Can we fix the typo in `comparision` (+ getter etc.) as well in the hotfix commit?

Reply:
Sure, and I found this typo also exists in `LogicalTypeJsonDeserializer` which will fail when users reload execution graph from json plan if we change it now. But luckily there's no user use it now, we can fix it.
Thus, it should be safe, and  I'll backport the hotfix commit to 1.13 too.